PROTOCOL.md

```diff
@@ -710,25 +807,39 @@ Writers should reject any transaction that contains data where the expression `x
 
 ## CHECK Constraints
 
+Enablement:
+- If the table is on a Writer Version up to 6, CHECK Constraints is enabled when the table's metadata contains required keys.
+- If the table is on Writer Version 7, a feature name `checkConstraints` must exist in the table `protocol`'s `writerFeatures`.
```

Comment:
The feature names are inconsistent in plural / singular. E.g. `checkConstraints` and `deletionVector` and `columnInvariants`. Let's go for plural?

Reply:
Yep, let's go for plural.